src/bootstrap/src/core/config/config.rs

```diff
@@ -1596,7 +1667,12 @@ impl Config {
                 );
                 exit!(2);
             });
-            toml.merge(included_toml, ReplaceOpt::IgnoreDuplicate);
+            toml.merge(
+                Some(t!(include_path.canonicalize())),
```

Comment:
This should be resolved relative to `toml_path`, otherwise if you run `x` from a different directory than the where the "root" config is, the include won't be resolved correctly.

Reply:
`include_path` is resolved from `config.src` which is not relative to the current working directory.

Reply:
Right, sorry, the include variable name confused me, this is the profile code. So only the include path should be changed (see other comment).